### PR TITLE
Vlad 14 k w to hp

### PR DESCRIPTION
**Pull request by umanskyi31**

### CODE REVIEW

File: php/Tests/ConverterTest.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        $this->assertEquals(3, Converter::convertKilowattsToHorsepower(2));
+        $this->assertEquals(5, Converter::convertKilowattsToHorsepower(4));
+        $this->assertEquals(8, Converter::convertKilowattsToHorsepower(6));
+        $this->assertEquals(13, Converter::convertKilowattsToHorsepower(10));
+        $this->assertEquals(194, Converter::convertKilowattsToHorsepower(145));
+    }
+
+    /**
+     * @covers Converter::convertKilowattsToHorsepower()
```
Comment (p5ych0) [inline php/Tests/ConverterTest.php]: replace with `@dataProvider` https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers everywhere

File: php/Utils/Converter.php [hunk 2]
```diff
@@ -25,7 +32,15 @@ public static function convertWattsToBTUPerHour(int $watts): int
      */
     public static function convertKilowattsToHorsepower(int $kW): int
     {
-        // @TODO: implement and cover with tests
+        try {
+            if(!is_int($kW) || $kW < 0) {
```
Comment (p5ych0) [inline php/Utils/Converter.php]: is_int is not required, values other than int will throw TypeError

File: php/Utils/Converter.php [hunk 3]
```diff
@@ -25,7 +32,15 @@ public static function convertWattsToBTUPerHour(int $watts): int
...
+                throw new \InvalidArgumentException('Value must be integer and not negative');
+            }
+            return round(self::HORSE_POWER * $kW);
+        } catch (\InvalidArgumentException $e) {
```
Comment (p5ych0) [inline php/Utils/Converter.php]: with this block the exception won't be thrown

Comment (vladumanskyi) [reply php/Utils/Converter.php]: Can the variable HP changed ? For example used not only mechanical HP. If yes, i can check the type variable.

Comment (p5ych0) [reply php/Utils/Converter.php]: let's assume it won't change

**Pull request closed without merging**